packages/charts/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -937,7 +941,9 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
       barHeight = Math.abs(barHeight);
       // Calculate threshold for minimum visible bar height
       const maxHeightFromBaseline =
-        this._yMax < 0 ? Math.abs(this._yMin - yReferencePoint) : Math.abs(this._yMax - yReferencePoint);
+        this._yMax < 0
+          ? Math.abs(this._yMin - yReferencePoint)
+          : Math.max(Math.abs(this._yMax - yReferencePoint), Math.abs(this._yMin - yReferencePoint));
```

Comment:
this can be moved to a function as it is used in multiple places.

Reply:
okay.